docs/docs/smart-contracts/PSP34/Extensions/mintable.md

```diff
@@ -7,17 +7,33 @@ This example shows how you can reuse the implementation of [PSP34](https://githu
 
 ## How to use this extension
 
-First, you should implement basic version of [PSP34](/smart-contracts/PSP34).
+First, you should implement basic version of [PSP34](../psp34.md).
 
 After you can just add implementation of PSP34Mintable via `#[openbrush::implementation(PSP34Mintable)]` attribute.
 
 ```rust
-use openbrush::contracts::psp34::extensions::mintable::*;
-
-impl PSP34Mintable for Contract {}
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
+
+#[openbrush::implementation(PSP34, PSP34Mintable)]
+#[openbrush::contract]
+pub mod my_psp34_mintable {
+    use openbrush::traits::Storage;
+
+    #[derive(Default, Storage)]
+    #[ink(storage)]
+    pub struct Contract {
```

Comment:
I'm not sure, but as I remember, there should be a field for psp34::metadata as well

Reply:
i think that is not necessary in mintable example

Reply:
yeah, i was thinking that it's metadata, sry